src/D2L.Security.OAuth2/Utilities/D2LHttpClient.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace D2L.Security.OAuth2.Utilities
+{
+	public class D2LHttpClient : ID2LHttpClient
+	{
+		/// <summary>
+		/// This class implements sync versions of HTTPClient methods.
+		/// TODO: Replace the sync methods which are currently using Task.Run(async verison); task.Wait();
+		/// With (probably) HttpWebRequest
+		/// </summary>

```

Comment:
I think this comment belongs more on the `D2LHttpClient` class, rather than `m_httpClient`?